netbox_librenms_plugin/templates/netbox_librenms_plugin/_interface_sync_content.html

```
@@ -332,6 +364,24 @@ <h5 class="modal-title" id="netboxOnlyInterfacesModalLabel">
                                         {% endif %}
                                     </td>
                                     <td>{{ interface.description|default:"-" }}</td>
+                                    {% if migrated_to_marker %}
+                                    <td class="text-end">
+                                        {% if migrated_to_winner %}
+                                        <button type="button"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)